templates/application.json

```diff
@@ -5,51 +5,42 @@
   "Parameters" : {
     "StackName": {
       "Type": "String",
-      "Description": "Name of the ElasticBeanstalk environment",
-      "Default" : "HyboxApplication"
```

Comment:
Should a default StackName value be set in defaults.json, just in case the command line parameter is not included on the create stack call?